src/main/java/com/ays/user/controller/UserSelfController.java

```diff
@@ -15,7 +15,7 @@
  * User Self Controller class for managing user's own operations.
  */
 @RestController
-@RequestMapping("/api/v1/user/self")
+@RequestMapping("/api/v1/user-self")
```

Comment:
Why did we change the endpoint?

Reply:
gives the impression that this and user controller are exactly the same, looks like even the service layer is different, so maybe create a distinct endpoint for this?